swan/python/jupyter_swan_config.py

```diff
@@ -0,0 +1,35 @@
+import os
+#from swancontents import get_templates
+
+c.FileCheckpoints.checkpoint_dir = os.environ.get("HOME") + "/.ipynb_checkpoints"
```

Comment:
The previous image did not store the checkpoints on EOS (HOME). Either way, our checkpoints class does not need this config and it makes it even more confusing. Please remove it.